packages/ckeditor5-engine/docs/framework/guides/deep-dive/custom-element-conversion.md

```diff
@@ -274,6 +270,75 @@ editor.conversion.for( 'dataDowncast' )
 	.add( dispatcher => dispatcher.on( 'insert:infoBox', dataDowncastConverter ) );
 ```
 
+### The model to view position mapping
```

Comment:
Don't we have to work on reverse direction mapping too? Or does it work out of the box? If so, it'd be good to mention that at the end of this section (e.g. in an info-box ... the Umberto's info-box :D)

Reply:
The view to model works ok because it uses `findMappedViewAncestor()`. Can you add it?